src/main/java/com/scottlogic/Main.java

```diff
@@ -34,4 +35,18 @@ private static void printAllUserPosts(List<UserPost> userPosts) {
             System.out.println("");
         }
     }
+
+    /**
+     * Prints out all user posts sorted by author ascending
+     * @param userPosts The list of user posts to sort then print out
+     */
+    private static void printUserPostsByAuthor(List<UserPost> userPosts) {
+        AuthorPostSorter authorPostSorter = new AuthorPostSorter();
+        List<UserPost> sortedPostsByAuthor = authorPostSorter.sort(userPosts);
+        System.out.println("Sorted List: ");
+        for(UserPost userPost : sortedPostsByAuthor) {
+            System.out.println(userPost);
+            System.out.println("");
+        }
+    }
```

Comment:
I'll allow print for now but professional code should use a logger,
Google SLF4J.
You haven't encountered annotations yet, but they save a lot of time.

Reply:
Duly noted!